subhub/api/webhooks/routes/salesforce.py

```diff
@@ -4,15 +4,9 @@
 
 from subhub.api.webhooks.routes.abstract import AbstractRoute
 from subhub.cfg import CFG
-from subhub import secrets
+from subhub import log
 
-
-logger = logging.getLogger("salesforce_route")
-log_handle = logging.StreamHandler()
-log_handle.setLevel(logging.INFO)
-logformat = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
-log_handle.setFormatter(logformat)
-logger.addHandler(log_handle)
+logger = log.setup(logging.getLogger("salesforce_route"))
```

Comment:
Why is this "salesforce_route" instead of "salesforce" like "firefox" is?

Reply:
Will update :)